src/Adapter/TaxRulesGroup/Repository/TaxRulesGroupRepository.php

```diff
@@ -40,64 +39,6 @@
  */
 class TaxRulesGroupRepository extends AbstractObjectModelRepository
 {
-    /**
-     * @var Connection
-     */
-    private $connection;
-
-    /**
-     * @var string
-     */
-    private $dbPrefix;
-
-    /**
-     * @param Connection $connection
-     * @param string $dbPrefix
-     */
-    public function __construct(
-        Connection $connection,
-        string $dbPrefix
-    ) {
-        $this->connection = $connection;
-        $this->dbPrefix = $dbPrefix;
-    }
-
-    /**
-     * @param TaxRulesGroupId $taxRulesGroupId
-     *
-     * @return array
-     */
-    public function getTaxRulesGroupDetails(TaxRulesGroupId $taxRulesGroupId): array
```

Comment:
if Im not mistaken this class was introduced in 1.7.8 with first beta of product page. So its all ok to remove those public methods, as for beta its not considered BC break, right @jolelievre ?

Reply:
well i removed it because its never used after i deleted and changed the part that was using it.

Reply:
If it's not used anymore it's fine to clean it since the code is still experimental